### DISCUSSION TITLE
Make error report more sense

### DISCUSSION THREAD

**Discussion by CsloudX** [godotengine/godot-proposals#10190] [Editor]
### Describe the project you are working on

No-Game App

### Describe the problem or limitation you are having in your project

For my project, it report error everytime when i open my project.
![image](https://github.com/user-attachments/assets/05b91ef5-8ad1-40a6-8d88-ed12ec0e41db)
but I don't know which file was error so i don't how to fix it.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

report which file was error, and i can to check it.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

none

### If this enhancement will not be used often, can it be worked around with a few lines of script?

no. i can't do anything.

### Is there a reason why this should be core and not an add-on in the asset library?

it should be core.

**Comment by Calinou** [comment]
Moving to discussion, as there is no concrete technical implementation provided.

Can you upload a minimal reproduction project somewhere?

Also, errors with cryptic messages should be reported in https://github.com/godotengine/godot/issues/42719. Don't open a proposal every time you stumble upon one; the tracker issue I linked is designed to host all reports of cryptic error messages.
